components/Sidebar/index.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="sidebar">
+    <div class="sidebar__logo">
+      <img width="133" height="25" src="~/assets/logo/logo-dedi-typo-small.svg" alt="Logo Desa Digital">
+      <img width="211" height="40" src="~/assets/logo/logo-text-cms.svg" alt="Logo text Content Management System">
+    </div>
+    <div class="sidebar__menu">
+      <nuxt-link v-for="menu in mainMenu" :key="menu.path" :to="menu.path" class="sidebar__menu-item">
+        <div class="sidebar__menu-item-box">
+          <!-- eslint-disable-next-line vue/no-v-html -->
+          <div class="sidebar__menu-item-icon" v-html="svg(menu.icon)" />
+          <span class="sidebar__menu-item-title">{{ menu.name }}</span>
+        </div>
+        <jds-icon
+          v-if="menu.arrow"
+          name="chevron-right"
+          size="16px"
+          class="sidebar__menu-item-arrow"
+        />
+      </nuxt-link>
+      <div class="sidebar__menu-title">
+        Master Data
+      </div>
+      <nuxt-link v-for="menu in masterDataMenu" :key="menu.path" :to="menu.path" class="sidebar__menu-item">
+        <div class="sidebar__menu-item-box">
+          <!-- eslint-disable-next-line vue/no-v-html -->
+          <div class="sidebar__menu-item-icon" v-html="svg(menu.icon)" />
```

Comment:
nice trick ❤️ , I often struggle to change Icon color because it's been converted to image when using `<img />` tag, I think this approach solve the problem  👍🏼 

is this approach only works on Nuxt with `nuxt/svg` package? @adzharamrullah

Reply:
❤️ 😃 
I really don't know if it's implemented in other frameworks other than nuxt, if you using VueJS try this library [vue-svg-loader](https://vue-svg-loader.js.org/)